### DISCUSSION TITLE
Clean version Option

### DISCUSSION THREAD

**Discussion by djmac84** [tidal-music/discussions#83] [Ideas & Feature requests]
Hi, My names Mac and I have a suggestion that would make it easier for us DJ's to find clean versions to songs. If we could right click an Explicit song and have an option to replace that current selected Explicit song(s) with the clean version. That would be revolutionary!! Also if we can do this by batch and group songs by Explicit and clean versions!! This would help so much! Especially in DJ software! Thank you!!

**Comment by stefanmoro** [comment]
Hi @djmac84 

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API/SDK, open source projects etc).

As this question relates to an existing product offering, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).

Thanks
